Source/WebInspectorUI/UserInterface/Views/RecordingActionTreeElement.js

```diff
@@ -112,6 +116,28 @@ WI.RecordingActionTreeElement = class RecordingActionTreeElement extends WI.Gene
             return parameterElement;
         }
 
+        function copyTextForElement(element, swizzleType) {
+            const textContext = element.textContent;
+            switch (swizzleType) {
+            case WI.Recording.Swizzle.WebGLBuffer:
+            case WI.Recording.Swizzle.WebGLFramebuffer:
+            case WI.Recording.Swizzle.WebGLRenderbuffer:
+            case WI.Recording.Swizzle.WebGLTexture:
+            case WI.Recording.Swizzle.WebGLShader:
+            case WI.Recording.Swizzle.WebGLProgram:
+            case WI.Recording.Swizzle.WebGLUniformLocation:
+            case WI.Recording.Swizzle.WebGLQuery:
+            case WI.Recording.Swizzle.WebGLSampler:
+            case WI.Recording.Swizzle.WebGLSync:
+            case WI.Recording.Swizzle.WebGLTransformFeedback:
+            case WI.Recording.Swizzle.WebGLVertexArrayObject:
+                // Strip-off trailing object id beginning at ':' to the end of the string
+                return textContent.replace(/:.*$/, "");
```

Comment:
rather than manipulate the string afterwards, i think it might be cleaner to have `createParameterElement` return a `{element, copyText}`
```
function createParameterUI(parameter, swizzleType, index) {
    let parameterCopyText = "";

    ...
    case WI.Recording.Swizzle.WebGLVertexArrayObject:
        parameterCopyText = WI.Recording.displayNameForSwizzleType(swizzleType);

        parameterElement.classList.add("swizzled");
        parameterElement.textContent = parameterCopyText + (parameter ? (`: (@${parameter})`) : "");
        break;
    ...

    if (! parameterCopyText.length)
        parameterCopyText = element.textContent;

    return {parameterElement, parameterCopyText};
}
```

and then use it like so
```
...
let {parameterElement, parameterCopyText} = createParameterElement(parameter, swizzleType, i);
parametersContainer.appendChild(parameterElement);

if (i)
    copyText += ", ";

copyText += parameterCopyText;
...
```